rllib/utils/minibatch_utils.py

```diff
@@ -140,6 +159,11 @@ def get_len(b):
                     n_steps -= len_sample
                     s = 0
                     self._num_covered_epochs[module_id] += 1
```

Comment:
This is actually the same logic like `independent` sampling mode in our `MultiAgentEpisodeBuffer`s. For the future we should reduce code again.

Reply:
Ah, great catch! We'll get rid of this iterator anyways b/c it relies on `MultiAgentBatch`. We should rewrite it to rely only on a plain dict.